Explorer/Assets/DCL/PluginSystem/Global/LandscapePlugin.cs

```diff
@@ -19,13 +19,15 @@ namespace DCL.PluginSystem.Global
 {
     public class LandscapePlugin : IDCLGlobalPlugin<LandscapeSettings>, ILandscapeInitialization
     {
-        private TerrainGenerator terrainGenerator;
+        public TerrainGenerator TerrainGenerator;
+        public WorldTerrainGenerator WorldTerrainGenerator;
+
         private readonly IAssetsProvisioner assetsProvisioner;
         private readonly IDebugContainerBuilder debugContainerBuilder;
         private ProvidedAsset<RealmPartitionSettingsAsset> realmPartitionSettings;
         private readonly MapRendererTextureContainer textureContainer;
         private readonly bool enableLandscape;
-        private ProvidedAsset<LandscapeData> landscapeData;
+        public ProvidedAsset<LandscapeData> landscapeData;
```

Comment:
Let's not do it: it's the connected to the previous comment: you can construct anything you need with the dependencies. But if you need them outside of the plugin - they probably should be a part of the container, not a plugin. Can you take a look? it's very cheaty...